release.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+vercomp () {
+	if [[ $1 == $2 ]]
+	then
+		return 0
```

Comment:
if the new version is the same as old version, we should error out, right?

Reply:
yup